docs/quickstart_contrib.rst

```diff
@@ -23,16 +23,14 @@ root of the project.
 
     $ python runtests.py
 
-You can limit the tests or pass paramaters as if you had called
-``manage.py test``.
+You can limit tests or pass paramaters as when using ``manage.py test``.
 
 .. code:: console
 
     $ ./runtests.py tests.test_basic -v 3
 
-If you have Python 3.4, 3.5, and 3.6 installed on your system, you may
-use ``tox`` to run all linters and test the package with multiple Python and
-Django versions
+If you have Python 3.4 or greater installed, you may use ``tox`` to run 
```

Comment:
Not quite: the meaning of the original sentence is that you must have Python 3.4 and Python 3.5 and Python 3.6 installed (and we should actually add Python 3.7 to that list). The reworked sentence means that you can have any of these installed, which is incorrect (`tox` will complain loudly when it cannot find one of those executables).